src/functions-and-arrays.js

```diff
@@ -52,14 +118,35 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
```

Comment:
Here you are `returning` the `first word` that `repeats` in the `array`. But what they ask you in the lab instructions is to `return` a new `array` without the `repeated words`. So what you need to do is create a `loop` that will run throughout all the `items` of the `array`  and `.push()` the items into the new `array` if that `word doesn't exist` in the new array yet:

```
function uniquifyArray(array) {

  if (array.length === 0){
    return null;
  }

  let uniqueArray = [];

  for (let i=0; i<array.length; i++){
    if (!uniqueArray.includes(array[i])){
      uniqueArray.push(array[i]);
    }
  }

    return uniqueArray;
}
```